shared/chat/conversation/header-area/normal/index.stories.tsx

```diff
@@ -28,6 +29,7 @@ const load = () => {
     .add('Username Header muted', () => <UsernameHeader {...defaultProps} muted={true} />)
     .add('Channel Header for small team', () => <ChannelHeader {...defaultProps} />)
     .add('Channel Header for big team', () => <ChannelHeader {...defaultProps} smallTeam={false} />)
+    .add('Phone header', () => <PhoneHeader {...defaultProps} participants={phones} />)
```

Comment:
You need to also change the prop provider so that it does the same thing `container.tsx` does.

https://github.com/keybase/client/blob/f85e66bde8535a3ec34d3aa021e1f2f3171b7bc1/shared/stories/prop-providers.tsx#L38-L46

Reply:
I'm confused - I did this work in `index` - the work I did in `container` doesn't need to be replicated in the provider since I'm specifically calling `PhoneHeader` here. Unless I'm misunderstanding?

Reply:
You're right, my bad.